advanced_bank_reconciliation/api/cash_coding.py

```
@@ -0,0 +1,209 @@
+			if account.account_type in ["Receivable", "Payable"] and (
+				not row.get("party_type") or not row.get("party")
+			):
+				frappe.throw(_("Contact is required for receivable or payable accounts."))
+			results.append(
+				{
+					"bank_transaction": transaction.name,
+					"status": "valid",
+					"message": _("Ready"),
+				}
+			)
+		except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)